Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -190,6 +190,17 @@ private string GetElemInfo(object o)
 
     public object ConvertToNative(Base @object)
     {
+      // schema check
+      var speckleSchema = @object["@SpeckleSchema"] as Base;
```

Comment:
I think we'll also need to add similar logic in the `CanConvertToNative` method so that it returns true if the `speckleSchema` prop is not null